docs/next.config.js

```diff
@@ -120,45 +138,6 @@ module.exports = withDocsInfra({
               },
             ],
           },
-          // transpile 3rd party packages with dependencies in this repository
-          {
-            test: /\.(js|mjs|jsx)$/,
-            resourceQuery: { not: [/raw/] },
-            include:
-              /node_modules(\/|\\)(notistack|@mui(\/|\\)x-data-grid|@mui(\/|\\)x-data-grid-pro|@mui(\/|\\)x-license-pro|@mui(\/|\\)x-data-grid-generator|@mui(\/|\\)x-date-pickers-pro|@mui(\/|\\)x-date-pickers|@mui(\/|\\)x-charts|@mui(\/|\\)x-tree-view)/,
```

Comment:
Yes, we were aliasing in babel, meaning we had to transpile these packages to apply the aliasing. Babel is the wrong place to do this, because it only operates on a subset of the files. You may end up with half your bundle with aliases applied and half without.

webpack acts as a linker, it connects all the imports together. When we alias a dependency in webpack it applies to every import, regardless of whether it ran through babel. It's a lot less error-prone to do it in this phase, and a lot less wasteful. 

If we weren't transpiling these files for any other reason than applying the alias, we can simply stop transpiling them.

Reply:
> If we weren't transpiling these files for any other reason than applying the alias, we can simply stop transpiling them.

I don't recall a reason to transpile npm dependencies other than to change the imports.